pkg/kubelet/rkt/rkt.go

```diff
@@ -72,17 +72,17 @@ const (
 	// the grace period from MinAge in ContainerGCPolicy.
 	//
 	// Duration to wait before discarding inactive pods from garbage
-	defaultGracePeriod = "1m"
+	defaultGracePeriod = "0"
 	// Duration to wait before expiring prepared pods.
-	defaultExpirePrepared = "1m"
+	defaultExpirePrepared = "0"
```

Comment:
@yujuhong Just for testing, I changed so I can see the result soonner. Changed back.
btw, we should derive this period from gc policy's MinAge, but that requires the gc definition stuff to be moved to kubelet/container or other packages, not in the kubelet.